dev-docs/modules/mediafilterRtdProvider.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page_v2
+title: The Media Trust Real-Time Ad Defense
+display_name: The Media Trust Real-Time Ad Defense
+description: Stop malware, redirects, scams, offensive ad content, heavy ads, and more from ruining your audience's experience.
+page_type: module
+module_type: rtd
+module_code : mediafilterRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# The Media Trust Real-Time Ad Defense
+
+## Overview
+
+ As malvertising, scams, and controversial and offensive ad content proliferate across the digital media ecosystem, publishers need advanced controls to both shield audiences from malware attacks and ensure quality site experience. With the market’s fastest and most comprehensive real-time ad quality tool, The Media Trust empowers publisher Ad/Revenue Operations teams to block a wide range of malware, high-risk ad platforms, heavy ads, ads with sensitive or objectionable content, and custom lists (e.g., competitors). Customizable replacement code calls for a new ad to ensure impressions are still monetized.
```

Comment:
Sorry, you have to tone down the marketing speak. Because Prebid cannot police who is "fastest" or "most comprehensive", you are not allowed to make this kind of claim. Please rewrite to something more like "A fast and comprehensive real-time ad quality tool..."

Reply:
Thanks @bretg, does this work better?

> As malvertising, scams, and controversial and offensive ad content proliferate across the digital media ecosystem, publishers need advanced controls to both shield audiences from malware attacks and ensure quality site experience. Via a speedy script, The Media Trust empowers publisher Ad/Revenue Operations teams to block malware, high-risk ad platforms, heavy ads, ads with sensitive or objectionable content, and custom lists (e.g., competitors)—all in real-time. Customizable replacement code calls for a new ad to ensure impressions are still monetized.